bakery/common_test.go

```diff
@@ -169,9 +169,5 @@ func basicAuthFromContext(ctx context.Context) (user, password string) {
 }
 
 func mustGenerateKey() *bakery.KeyPair {
```

Comment:
Do you really need this function at all now?

Reply:
mustGenerateKey is a bit more wieldy than bakery.MustGenerateKey in tests, so I decided to leave it around for now.